scripts/run_notebooks.py

```diff
@@ -0,0 +1,55 @@
+"""Script to run all notebooks example notebooks.
+
+Taken from: https://github.com/pymc-labs/pymc-marketing/blob/main/scripts/run_notebooks/runner.py
+"""
+
+import logging
+from pathlib import Path
+
+import papermill
+from joblib import Parallel, delayed
+from tqdm import tqdm
+
+KERNEL_NAME: str = "python3"
+DOCS = Path("docs")
+NOTEBOOKS: list[Path] = [
+    # DOCS / "compare-fixest-pyfixest.ipynb",  # needs R
```

Comment:
Let me work on this notebook on a different PR as it requires some investigation.

Reply:
The error likely stems from the fact that Stargazer's default does not have pyfixest support - makes sense to handle this in another pr!